  Header header
  uint32 message_id
  uint8 status
  uint8 motor_id
